### DISCUSSION TITLE
手机查看可能性探讨

### DISCUSSION THREAD

**Discussion by TheXIE383** [xfgryujk/blivechat#238] [提问] [answered]
有没有可以用手机浏览器打开的网页，方便用手机看弹幕

**Comment by xfgryujk** [comment, marked as answer]
https://github.com/xfgryujk/blivechat/wiki/%E5%A6%82%E4%BD%95%E5%9C%A8%E5%85%A8%E5%B1%8F%E6%B8%B8%E6%88%8F%E6%97%B6%E7%9C%8B%E5%BC%B9%E5%B9%95#%E4%BA%94%E7%94%A8%E6%89%8B%E6%9C%BA%E8%BF%9E%E6%8E%A5%E6%9C%AC%E5%9C%B0%E7%9A%84blivechat

**Comment by xfgryujk** [comment]
我都不知道怎么能问出来的，手机浏览器打开公共服务器也是一样的

![Screenshot 2024-10-01 212524](https://github.com/user-attachments/assets/0806a9d8-30f6-444e-b033-85bc7bdd97b4)

**Reply by TheXIE383** [reply]
主要是这个方法我试过，手机跟电脑连同一个网发现IPv4地址是不一样的，浏览器公共服务器扫码也是显示网络错误，我想问是哪里有问题
![image](https://github.com/user-attachments/assets/6d35e976-4031-4a94-bb53-cf4a1eb76c22)
![IMG_4870(20241002-193835)](https://github.com/user-attachments/assets/32611279-e8e1-4d8c-9783-0cd27f2f75d5)
![IMG_4869(20241002-193450)](https://github.com/user-attachments/assets/91659242-c13b-4f4e-84be-3ef076974216)

**Reply by xfgryujk** [reply]
你这看起来像是校园网啊。有的路由器有无线隔离的功能，两个无线设备不能直连，但是一个设备有线就可以。不是同一个网络也可以试试，只要能路由到就没问题。手机连公共服务器可能也要翻墙

**Reply by TheXIE383** [reply]
回去问了一下教我ICS的信院老师，他说是路由器跟防火墙的问题，他让我把手机跟电脑的IPv4路由器设置成一样的就好了
(虽然ICS也有上过两节网络，但因为没有相应的实验就没认真听)
![IMG_4874](https://github.com/user-attachments/assets/f892df8d-d891-49de-a78a-91e1a5afc1ab)
